docs/src/main/sphinx/connector/faker.md

```diff
@@ -0,0 +1,225 @@
+# Faker connector
+
+The Faker connector generates random data matching a defined structure. It uses
+the [Datafaker](https://www.datafaker.net/) library, which makes the generated
+data more realistic. It can be used to easily populate another data source with
+large and realistic test data, which can help with testing performance of
+applications processing data (like Trino itself), and also application's user
+interface.
+
+## Usage
+
+Faker generates data when reading from a table created in a catalog using this
+connector. This makes it easy to fill an existing schema with random data, by
+copying only the schema into a Faker catalog, and inserting the data back into
+the original tables.
```

Comment:
Finish this paragraph with something like

Using the catalog definition from Configuration you can proceed with the following steps: